deploy/olm-catalog/kubefed-operator/kubefed-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 packageName: kubefed-operator
 channels:
 - name: alpha
-  currentCSV: kubefed-operator.v0.1.0
```

Comment:
I don't think we need this change. The changes in art.yaml are to replace the currentCSV with the productized CSV?

Reply:
We _should_ actually remove this line - since what is going into the 4.2 branch will be the first productized version and will be distinct from the community versions.

Reply:
We need to keep this line as we're using that information in olm-e2e testing (https://github.com/openshift/kubefed-operator/blob/master/hack/catalog.sh#L17)
I changed it in the first place to ensure that we're testing changes against the latest version of downstream CSV.

Reply:
As discussed with @abhat, I'm keeping the version as-is i.e. `0.1.0`